### PR TITLE
Literate all examples

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

**Comment by navidcy** [comment]
Probably we should include some figures in the examples...

**Comment by navidcy** [comment]
@glwagner, can you confirm that docs can be build properly? You should be able to build the docs simply by calling
```bash
julia --project=docs/ -e 'using Pkg; Pkg.instantiate(); Pkg.develop(PackageSpec(path=pwd()))'
julia --project=docs/ docs/make.jl
```
from the repo directory.

File: examples/twodnavierstokes_stochasticforcing.jl [hunk 1]
```diff
@@ -25,33 +45,55 @@ x, y = gridpoints(gr)
 Kr = ArrayType(dev)([ gr.kr[i] for i=1:gr.nkr, j=1:gr.nl])
 
 force2k = @. exp(-(sqrt(gr.Krsq)-kf)^2/(2*dkf^2))
-force2k[gr.Krsq .< 2.0^2 ] .= 0
-force2k[gr.Krsq .> 20.0^2 ] .= 0
+force2k[gr.Krsq .< 2.0^2 ] .= 0  # making sure that focing has no power for low wavenumbers
+force2k[gr.Krsq .> 20.0^2 ] .= 0 # making sure that focing has no power for high wavenumbers
```
Comment (glwagner) [inline examples/twodnavierstokes_stochasticforcing.jl:49]: why is it called `force2k`?

File: examples/twodnavierstokes_stochasticforcing.jl [hunk 2]
```diff
@@ -68,24 +110,16 @@ function makeplot(prob, diags)
   sca(axs[3]); cla()
 
   i₀ = 1
-  dEdt = (E[(i₀+1):E.i] - E[i₀:E.i-1])/cl.dt
+  dEdt = (E[(i₀+1):E.i] - E[i₀:E.i-1])/cl.dt #numerical approximation of energy tendency
```
Comment (glwagner) [inline examples/twodnavierstokes_stochasticforcing.jl:113]: You could be even more specific and say its a first-order approximation

File: examples/twodnavierstokes_stochasticforcing.jl (hunk 1 above)
Comment (navidcy) [reply examples/twodnavierstokes_stochasticforcing.jl:49]: fair point; resolved by 22704c4

**Comment by navidcy** [comment]
@glwagner, wanna have another look before I merge? I pushed more examples..

**Pull request merged**